labgrid/resource/udev.py

```diff
@@ -47,8 +46,7 @@ def poll(self):
             self.log.debug("%s: %s", device.action, device)
             for resource in self.resources:
                 self.log.debug(" %s", resource)
-                if resource.try_match(device):
-                    break
+                resource.try_match(device)
```

Comment:
`try_match()` can be simplified by turning `return False` into a `return`.

Reply:
I guess we can also replace the whole content of the`if self.device is None:` code block with a `return`, right?

Reply:
On second thought: do we really want to change `try_match()` at all? Maybe we want replace the break by a log message. If a match was found could still be interesting there.

Reply:
Yes, sounds good to me, please do so.

Reply:
Done, please take another look.